### DISCUSSION TITLE
increase memory usage after google drive addon backup in HA VM

### DISCUSSION THREAD

**Discussion by aperezva** [tteck/Proxmox#3357] [Q&A]
Hi,

It seems everything is working fine, but when I execute a backup ina HAOS VM in proxmox , memory usage increase dramatically. After backup some memory is free again, but day by day memory usaga increase and after 5 days without shutdown the VM mem usage is about 80%.

I have 16 Gb in the VM, and I don´t know what to clean memory after backups.

Another question.

If I m executing backups in proxmox, is it necessary to execute schedule  google driv e backups?

Thanks

**Comment by Mellowlynx** [comment]
A bit of more RAM usage is fine, but a dramatic increase not.

I would look at `top` thought the shell while starting a backup to see that happens on the system and what service is using the RAM.
You can do this by opening the HA VM shell, type `login` so you get `#` and then `top`.
Start the backup and check top to see what's happening. This might give you a clue.

"execute schedule google driv e backups?"
If you mean, if it's needed to export your PVE backups, you can… but you don't have to.
It depends on your situation. My backups are saved on a NAS, but I also sync the backups to Acronis Cloud, so I have an off-site backup.

If you main something else, it's always good to have an extra backup, especially if it doesn't cost you anything more.

**Comment by aperezva** [comment]
N
![IMG_1209](https://github.com/tteck/Proxmox/assets/58134028/83ab134d-9fc1-4512-8ec4-810092b9b755)


This is the increase but if I monitored inside Home assistant memory is not more than 13% but in proxmox is about 65%

I m lost.

**Comment by Mellowlynx** [comment]
It's normal to see that the active RAM on a VM or container is less than what is reported as “active” by a hypervisor like Proxmox.
In the LXC you could also see that there is some reserved/cache. This is included in the “active” RAM that PVE reports.

Here is my HomeAssistant RAM usages right now.
PVE says that 2.63 GB is in use.
![afbeelding](https://github.com/tteck/Proxmox/assets/2348339/8ad5c624-5885-40bd-9399-c6df432d078e)

In HA OS, in use is only 897 MB this is in use, and 1.8 GB is reserved/cache.
![afbeelding](https://github.com/tteck/Proxmox/assets/2348339/3d7033a0-5aaf-40bc-9793-230a6094b620)

Together, it's about the same that PVE shows.

**Reply by aperezva** [reply]
woow 

Understood.

The same to me. Now i understand.


In my case 2,3 Gb useb by HAOS and 8,5 GB in cache

In this case, I understand when HAOS it´s backuing up it will fix the cache memory, and only unblock if systems needs

Thanks a lot  very helpful

Last question. If I´m doing backup thru Proxmox, wiht BPS and also in a NAS, do i need to have anoteher backup exceuted with the google drive addon inside HAOS?

Thansk again
